var/spack/repos/builtin/packages/libunwind/package.py

```diff
@@ -62,10 +64,10 @@ class Libunwind(AutotoolsPackage):
 
     # The libunwind releases contain the autotools generated files,
     # but the git repo snapshots do not.
-    depends_on('autoconf', type='build', when='@master,1.5-stable')
-    depends_on('automake', type='build', when='@master,1.5-stable')
-    depends_on('libtool',  type='build', when='@master,1.5-stable')
-    depends_on('m4',       type='build', when='@master,1.5-stable')
+    depends_on('autoconf', type='build', when='@1.5-stable,1.6-stable:')
+    depends_on('automake', type='build', when='@1.5-stable,1.6-stable:')
+    depends_on('libtool',  type='build', when='@1.5-stable,1.6-stable:')
+    depends_on('m4',       type='build', when='@1.5-stable,1.6-stable:')
```

Comment:
Why was master removed from the list?

Reply:
@alalazo It's not removed.  Look closely, it says `1.6-stable:` with the colon, 
and master > 1.6-stable.

Reply:
@alalazo No, it does not include the point releases.
A couple weeks ago, I added `stable` as an infinity version.
So, 1.6.2 < 1.6-stable < master.

For example, `spack info` reports:
```
Safe versions:  
    master        [git] https://github.com/libunwind/libunwind on branch master
    1.6-stable    [git] https://github.com/libunwind/libunwind on branch v1.6-stable
    1.6.2         http://download.savannah.gnu.org/releases/libunwind/libunwind-1.6.2.tar.gz
    1.5-stable    [git] https://github.com/libunwind/libunwind on branch v1.5-stable
    1.5.0         http://download.savannah.gnu.org/releases/libunwind/libunwind-1.5.0.tar.gz
```

Reply:
In part, I added `stable` specifically for libunwind.
Their branches are named stable, so I wanted the spack name to match.

Reply:
Ugh, so there are 3 gotchas that people need to know to understand this:
1. `master` is an infinity version
2. `stable` is an infinity version
3. Spack versioning system treats `.` as it was `-` when generating the tuple used for comparison

Can we go back to having only fixed releases in this list, so we don't require users to know the above gotchas when trying to understand what will have `m4` & Co. as build dependencies?